guidelines.md

```diff
@@ -101,3 +101,61 @@ As an example, consider minimum and maximum length of the reservation:
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
 
+# Mews Certification
+
+Before going live in the production environment all connections must be certified by Mews.
+
+### Pre-Certification
+
+- Sign in to the Mews Connector API demo account. Credentials for this can be found in the [guidelines](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) section.
+- Create a unique access token. The client token will always be `E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D`.
+  - Go to **Main menu** > **Settings** > **Marketplace**.
+  - Search for **connector API Demo client**.
+  - You can search the Marketplace by using either the categories section (the application you need is in the Connectivity section), or by pressing Ctrl (Windows) or Command (Mac) + F to search by name.
+  - Select **see more** > **ok**.
+    - If you're having trouble, check the [Mews guide](https://help.mewssystems.com/reference#about-marketplace) for more information on setting up the connection.
+    - At this point, you'll now be viewing the active connections for the demo environment. Select Settings for the connection you've just created. 
+    - **Name:** name of your application/company.
+    - Select the **Enabled** checkbox, and click **Save**.
+      - If the connection is not enabled, you will not receive any data from the property.
+  - Add the unique access token (from the first step) to your code.
+
+### Certification Procedure
+
+All testing must be conducted in the [demo environment](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) before going live.
+
+Once you have completed building and testing the connection, send an email to marketplace@mewssystems.com with the following information:
+
+- Name of application.
+- Type of application (e.g. Revenue management, Point of sales etc.)
+- A list of all the intended API endpoints used, as well as the logic (use case) for each endpoint.
```

Comment:
Why aren't all of the necessary information in the form and half of it has to be provided via email and half via form?

Reply:
As Honza writes, and we discussed on Friday, everything should be in the application form.

Reply:
The items have been migrated to the application form and description has been adjusted accordingly.